docs/contribute.md

```diff
@@ -1,111 +1,35 @@
----
----
-# איך לתרום לפרויקט
-
----
-[עמוד הבית](https://linksforisrael.com/)
-[קבוצת הואצאפ שלנו](<https://chat.whatsapp.com/JjD8eijWfDXD10QbM2VyaX>)
 
 ## 🚀 Getting Started
 
-1. **Set Up Your Environment**
-   - Make sure you have Git and Ruby installed.
-
-2. **Fork the Repository**
-   - Go to our [Github Repo](https://github.com/4tal/LinksForIsrael).
-   - Click on the `Fork` button at the top right corner. This will create a copy of the repository in your own GitHub account.
+1. **Prerequisites**
+   - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+   - [Ruby 3](https://www.ruby-lang.org/en/documentation/installation/)
 
-3. **Clone the Forked Repository**
+1. **Clone the Forked Repository**
 
    ```bash
    git clone https://github.com/YOUR_USERNAME/LinksForIsrael.git
    cd LinksForIsrael
    ```
 
-4. **(Optional: If you want to run the GitHub Pages project locally) Install Dependencies**
-   - Install bundler:
-
-     ```bash
-     gem install bundler
-     ```
-
-   - Install project dependencies:
-
-     ```bash
-     bundle install
-     ```
-
-5. **Run Locally**
-
-   ```bash
-   bundle exec jekyll serve
-   ```
-
-   > ⚠️ **Note**: There is currently an issue with running the project locally. The theme doesn't display as expected. If you can solve this issue, your contribution would be greatly appreciated!
+1. **Install Dependencies**
+   - Install bundler: `gem install bundler`
+   - Install project dependencies: `bundle install`
 
-6. **Install Ruby 3**
-   Check how to install an up-to-date version of Ruby for your system [here](https://www.ruby-lang.org/en/documentation/installation/).
+1. ** Setup GitHub API token **
```

Comment:
Why do you need the token?
I didn't do any of these steps and it worked for me - maybe mark them as optional (and specify for which features/environments they are required)

Reply:
Without this, you are getting a warning: `GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.` Warnings are bad, and for issues that might be caused by this specific one (even if you didn't encounter them, or didn't notice them) see https://github.com/github/pages-gem/issues/399. Resolving this warning is a 1-minute process, so I vote to be safe rather than sorry.

Reply:
I had security in mind when I suggested to issue a token with readonly permissions.
But sure, I've made it optional and added the ticket for more information.